src/components/upcoming-event.js

```diff
@@ -0,0 +1,18 @@
+import EventsWindow from "./events-window";
+import styles from "@/styles/UpcomingEvent.module.css";
+
+export default function UpcomingEvent({ title, date, time, location }) {
+  return (
+    <>
+      <EventsWindow location={location} color="#65C7CC">
+        <div className={styles.container}>
+          <h2>{title}</h2>
+          <div className={styles.dateTime}>
+            <p>{date}</p>
+            <p>{time}</p>
```

Comment:
Could you update the font, font size, font color, and background color of the event location to more closely match the Figma?
![image](https://github.com/IllinoisWCS/IllinoisWCS.github.io/assets/80798381/5972f39d-f0d1-47d2-b3f6-577f74d7c136)